tests/algorithms/test_rand.py

```diff
@@ -0,0 +1,12 @@
+import pytest
+import pydp as dp
+
+
+def test_rand_UniformDouble():
+    ud = dp.UniformDouble()
+    assert isinstance(ud, float)
```

Comment:
https://github.com/google/differential-privacy/blob/master/differential_privacy/algorithms/rand_test.cc#L52
Can we add all of these tests?

Reply:
Yeah good idea, I'm putting an isinstance assertion for every instance and function we currently have binded. I figure it's a decent sanity check on return policies.

Reply:
While with these tests, we get to know that at least it is returning something sensible, we want to make sure that it doesn't return a junk value, hence tests for actual values.